orchagent/routeorch.cpp

```diff
@@ -1187,3 +1325,80 @@ bool RouteOrch::removeRoute(sai_object_id_t vrf_id, const IpPrefix &ipPrefix)
 
     return true;
 }
+
+bool RouteOrch::createRemoteVtep(sai_object_id_t vrf_id, const NextHopKey &nextHop)
+{
+    SWSS_LOG_ENTER();
+    EvpnNvoOrch* evpn_orch = gDirectory.get<EvpnNvoOrch*>();
+    VxlanTunnelOrch* tunnel_orch = gDirectory.get<VxlanTunnelOrch*>();
+    bool status = false;
+    int ip_refcnt = -2;
```

Comment:
Why is this initialized to -2? Can we have it as 0?

Reply:
Ok. Will do. Used it for debug purpose.